spec/return_codes_spec.rb

```diff
@@ -34,6 +34,7 @@
         end
 
         it "prints a message to STDERR" do
+          skip "fails on truffleruby" if RUBY_ENGINE == "truffleruby" && command.include?("testunit_bad.rb")
```

Comment:
@eregon could you specify why it fails on TruffleRuby, what's broken, where is the bug tracked if it is tracked?

Reply:
I filed https://github.com/oracle/truffleruby/issues/3535 and I'll add a reference to that here then.

Reply:
Done.